create_pdfs.py

```diff
@@ -119,8 +109,7 @@ def main():
             )
             title = None
             if args.assign_title:
```

Comment:
I ran into issues with this code too.  I didn't assume to change it since I was technically only editing code in the if-block.  In the future, I should have fixed this since I was updating the 'title' code.  But not fix the code directly below for the grade arg that follows the same code paradigm.

I have an update in the next PR coming soon.